wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -326,6 +329,9 @@ public void removeWorkspace(String workspaceId) throws ConflictException, Server
         if (runtimes.hasRuntime(workspaceId)) {
             throw new ConflictException("The workspace '" + workspaceId + "' is currently running and cannot be removed.");
         }
+
+        workspaceProjectStorageCleaner.remove(workspaceId);
```

Comment:
If it is clearer then method name `clear` looks more natural for me than `remove`